handbook/business-operations/business-operations.rituals.yml

```diff
@@ -174,4 +174,14 @@
   description: "Provide information to tax team with Deloitte and assist with filing and paying state and federal returns" 
   moreInfoUrl: 
   dri: "hollidayn"
+  -
```

Comment:
Hi @ireedy, this dash not being lined up with the rest of them on this page is causing this PR to fail the "Test Fleet website"

![image](https://github.com/fleetdm/fleet/assets/108141731/c8e9db04-5902-409d-8308-6973f4f7c4dc)

If you click `Details` you'll get a screen like this: 

![image](https://github.com/fleetdm/fleet/assets/108141731/c3d0bf6f-7da4-4e6b-a47f-9e7c1e7d72d7)

That will tell you what responsibility has the problem.